embedded-elasticsearch/src/main/java/com/playtika/test/elasticsearch/ElasticSearchProperties.java

```diff
@@ -30,6 +30,6 @@ public ElasticSearchProperties() {
     // https://hub.docker.com/_/elasticsearch
     @Override
     public String getDefaultDockerImage() {
-        return "docker.elastic.co/elasticsearch/elasticsearch:7.14.0";
+        return "docker.elastic.co/elasticsearch/elasticsearch:8.5.3";
```

Comment:
Spring Data Elasticsearch 5.0 [upgrades to Elasticsearch 8.5](https://docs.spring.io/spring-data/elasticsearch/docs/current/reference/html/#new-features.5-0-0).